events/tpcreaction.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    name: 'messageCreate',
+    once: false,
+    execute (message) {
+  if (message.thread?.name.startsWith('SCREENSHOT CONTEST') || message.channel.id === '830210202464813056') {
+    const image = message.attachments.find(attachment => attachment.contentType?.startsWith('image/'));
+    if (image) {
+      await message.react(':tpc:845075689241051138');

```

Comment:
Why are we blocking execution for the reaction here? I'm fairly certain you can just call `message.react()`. Alternatively, stop using `async/await`, and just use callbacks.